### PR TITLE
zh: update deploy-a-dm-cluster-using-binary for 2.0

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed, added or deleted? (Required)
zh: update deploy-a-dm-cluster-using-binary.md for 2.0

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 
preview: https://github.com/GMHDBJD/docs-dm/blob/deploy-a-dm-cluster-using-binary-2.0/zh/deploy-a-dm-cluster-using-binary.md

### CODE REVIEW

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 1]
```diff
@@ -98,112 +159,71 @@ DM-worker 的配置文件：
...
-        打印出 DM-master 的示例配置
+fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg=
 ```
 
-> **注意：**
->
-> 某些情况下，无法使用命令行参数的方法来配置 DM-worker，因为有的配置并未暴露给命令行。
-
-**配置方式 2：配置文件**
+记录该加密后的值，用于下面 MySQL1 的配置。把以下配置文件内容写入到 `conf/source1.toml` 中。
```
Comment (toutdesuite) [inline zh/deploy-a-dm-cluster-using-binary.md:199]:
```suggestion
记录该加密后的密码，用于下面 MySQL1 的配置。把以下配置文件内容写入到 `conf/source1.toml` 中。
```

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 2]
```diff
@@ -98,112 +159,71 @@ DM-worker 的配置文件：
...
 
-推荐使用配置文件，把以下配置文件内容写入到 `conf/dm-master.toml` 中。
+MySQL1 的配置文件
 
-DM-master 的配置文件：
+如需要启用 GTID 模式，则额外配置 enable-gtid = true。
```
Comment (toutdesuite) [inline zh/deploy-a-dm-cluster-using-binary.md:203]:
```suggestion
如需要启用 GTID 模式，则额外配置 enable-gtid 为 true。
```

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 3]
```diff
@@ -98,112 +159,71 @@ DM-worker 的配置文件：
...
+ 
+[from]
+host = "192.168.0.1"
+user = "root"
+password = "fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg="
+port = 3306
 ```
 
-在终端中使用下面的命令运行 DM-master：
+在终端中使用下面的命令，使用 dmctl 将数据源配置加载到 DM 集群中 ：
```
Comment (toutdesuite) [inline zh/deploy-a-dm-cluster-using-binary.md:217]:
```suggestion
在终端中执行下面的命令，使用 dmctl 将数据源配置加载到 DM 集群中 ：
```

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 4]
```diff
@@ -15,38 +15,105 @@ category: how-to
 
 ## 使用样例
 
-假设在两台服务器上部署 MySQL，在一台服务器上部署 TiDB（mocktikv 模式），另外在三台服务器上部署两个 DM-worker 实例和一个 DM-master 实例。各个节点的信息如下：
+假设在两台服务器上部署 MySQL，在一台服务器上部署 TiDB（mocktikv 模式），另外在四台服务器上部署两个 DM-worker 实例和两个 DM-master 实例。各个节点的信息如下：
```
Comment (WangXiangUSTC) [inline zh/deploy-a-dm-cluster-using-binary.md:18]: 这里部署三个 dm-master 比较好一点？

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 5]
```diff
@@ -98,112 +159,71 @@ DM-worker 的配置文件：
...
-> 某些情况下，无法使用命令行参数的方法来配置 DM-worker，因为有的配置并未暴露给命令行。
-
-**配置方式 2：配置文件**
+记录该加密后的密码，用于下面 MySQL1 的配置。把以下配置文件内容写入到 `conf/source1.toml` 中。
 
-推荐使用配置文件，把以下配置文件内容写入到 `conf/dm-master.toml` 中。
+MySQL1 的配置文件
 
-DM-master 的配置文件：
+如需要启用 GTID 模式，则额外配置 enable-gtid 为 true。
```
Comment (WangXiangUSTC) [inline zh/deploy-a-dm-cluster-using-binary.md:203]: enable-gtid 写到下面的配置文件中吧，可以在注释中说明

**Comment by toutdesuite** [comment]
@yikeke PTAL. Thanks

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 6]
```diff
@@ -98,112 +160,72 @@ DM-worker 的配置文件：
...
+ 
+[from]
+host = "192.168.0.1"
+user = "root"
+password = "fCxfQ9XKCezSzuCD0Wf5dUD+LsKegSg="
+port = 3306
 ```
 
-在终端中使用下面的命令运行 DM-master：
+在终端中执行下面的命令，使用 dmctl 将数据源配置加载到 DM 集群中 ：
```
Comment (toutdesuite) [inline zh/deploy-a-dm-cluster-using-binary.md:219]:
```suggestion
在终端中执行下面的命令，使用 dmctl 将数据源配置加载到 DM 集群中：
```

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 7]
```diff
@@ -15,38 +15,106 @@ category: how-to
...
+
+在终端中使用下面的命令运行 DM-master：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+./bin/dm-master -config conf/dm-master1.toml
+```
+
+对于 DM-master2 和 DM-master3 ，修改配置文件中的 `name` 为 `master2` 和 `master3` ，并将 `peer-urls` 的值改为 `192.168.0.5:8291` 和 `192.168.0.6:8291` 即可。
```
Comment (yikeke) [inline zh/deploy-a-dm-cluster-using-binary.md:111]:
```suggestion
对于 DM-master2 和 DM-master3，修改配置文件中的 `name` 为 `master2` 和 `master3`，并将 `peer-urls` 的值改为 `192.168.0.5:8291` 和 `192.168.0.6:8291` 即可。
```

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 8]
```diff
@@ -98,112 +160,72 @@ DM-worker 的配置文件：
...
-对于 DM-worker2，修改配置文件中的 `source-id` 为 `mysql-replica-02`，并将 `from` 配置部分修改为 MySQL2 的地址即可。如果因为没有多余的机器，将 DM-worker2 与 DM-worker1 部署在一台机器上，需要把两个 DM-worker 实例部署在不同的路径下，否则保存元信息和 relay log 的默认路径会冲突。
+对于 DM-worker2，修改配置文件中的 `name` 为 `worker2` 即可。
 
-### DM-master 的部署
-
-DM-master 提供命令行参数和配置文件两种配置方式。
+### 配置 MySQL 数据源
 
-**配置方式 1：命令行参数**
+运行数据同步任务前，需要对 source 进行配置，也就是 MySQL 的相关设置。且为了安全，强制用户配置加密后的密码。首先使用 dmctl 对 MySQL 的密码进行加密，以密码为 "123456" 为例：
```
Comment (yikeke) [inline zh/deploy-a-dm-cluster-using-binary.md:188]:
```suggestion
运行数据同步任务前，需要对 source 进行配置，也就是 MySQL 的相关设置。且为了安全，强制用户配置及使用加密后的密码。首先使用 dmctl 对 MySQL 的密码进行加密，以密码为 "123456" 为例：
```

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 9]
```diff
@@ -98,112 +160,72 @@ DM-worker 的配置文件：
...
->
-> 某些情况下，无法使用命令行参数的方法来配置 DM-worker，因为有的配置并未暴露给命令行。
-
-**配置方式 2：配置文件**
+记录该加密后的密码，用于下面 MySQL1 的配置。把以下配置文件内容写入到 `conf/source1.toml` 中。
 
-推荐使用配置文件，把以下配置文件内容写入到 `conf/dm-master.toml` 中。
-
-DM-master 的配置文件：
+MySQL1 的配置文件
```
Comment (yikeke) [inline zh/deploy-a-dm-cluster-using-binary.md:202]:
```suggestion
MySQL1 的配置文件：
```

File: zh/deploy-a-dm-cluster-using-binary.md [hunk 10]
```diff
@@ -98,112 +160,72 @@ DM-worker 的配置文件：
...
+在终端中执行下面的命令，使用 dmctl 将数据源配置加载到 DM 集群中：
 
 {{< copyable "shell-regular" >}}
 
 ```bash
-bin/dm-master -config conf/dm-master.toml
+./bin/dmctl --master-addr=192.168.0.4:8261 operate-source create conf/source1.toml
 ```
 
+对于 MySQL2 ，修改配置文件中的 `name` 为 `mysql-replica-02`， `host` 为 `192.168.0.2`，并将 `password` 和 `port` 改为相应的值即可。
```
Comment (yikeke) [inline zh/deploy-a-dm-cluster-using-binary.md:227]:
```suggestion
对于 MySQL2，修改配置文件中的 `name` 为 `mysql-replica-02`，`host` 为 `192.168.0.2`，并将 `password` 和 `port` 改为相应的值即可。
```

**Pull request merged**